src/components/ItemFilter.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 
-export const ItemFilter = ({ item }) => {
+export const ItemFilter = ({ element, selectedElement }) => {
+  const selectedStyles = selectedElement === element.en
+    ? {
+      color: 'white',
+      backgroundColor: '#11698E',
+      paddingLeft: '0.5rem',
+    }
+    : {
+      paddingLeft: '0.5rem',
+    };
   return (
-    <div className='py-2 ml-2 cursor-pointer text-black hover:no-underline'>{item}</div>
+    <div style={selectedStyles} className='py-2 cursor-pointer text-black hover:no-underline'>{element.es}</div>
```

Comment:
You can use the TailwindCSS utilities classes to define padding left size if item is active or if is not active, instead of handle it using style

Tailwind Classes | properties 
-- | --
pr-2 | padding-right: 0.5rem; /* 8px */
pb-2 | padding-bottom: 0.5rem; /* 8px */
pl-2 | padding-left: 0.5rem; /* 8px */



```suggestion
    <div style={selectedStyles} className='py-2 cursor-pointer text-black hover:no-underline pl-2'>{element.es}</div>
```